stdlib/public/core/Integers.swift

```diff
@@ -1329,6 +1330,13 @@ extension BinaryInteger {
     return self > 0 && self & (self - 1) == 0
   }
 
+  @usableFromInline @_transparent
```

Comment:
I don't know if you actually want all these to be `transparent` rather than `inlinable`. Good for constants, potentially bad for non-constants.

Reply:
Yes, it should be `inlinable` here. Thank you for pointing this out!